microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -10,13 +10,14 @@ ms.date: 10/24/2022
 ---
 # Windowed vs. visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+There are three options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting, window to visual hosting, and visual hosting.
```

Comment:
consider list format - coordinate with greatly condensed equivalent of table, summarized to just the essence (1 phrase each).  eg:
* foo - does x.
* bar - does x.
* baz - does x.

Reply:
I can list the 3 hosting modes, but I like the follow up paragraph giving a readers an opt-out in case this info isn't necessary for them (assuming they're trying to start out quickly). That feels awkward with the bullet+explainer suggestion.

Reply:
result:
```
There are three options for hosting the Microsoft Edge WebView2 control in your app:
* Windowed hosting.
* Window to Visual hosting.
* Visual hosting.
```